test/unit/test.natural_language_understanding.v1.js

```diff
@@ -51,4 +51,13 @@ describe('natural_language_understanding', function() {
 
     nlu.analyze(options, done);
   });
+
+  it('should list models', function(done) {
+    nock(watson.NaturalLanguageUnderstandingV1.URL)
+      .persist()
```

Comment:
I know the other one is wrong also, but `.persist()`shouldn't be called on nock's that are set up for a single test. (Actually, can you remove the other `.persist()` while you're at it?)